extending-valtimo/document/configuring-search-fields.md

```diff
@@ -40,7 +40,7 @@ should have the following structure:
 If a specific search field is already present in the database this field is not overwritten. It was probably changed by an
 administrator in the user interface, these changes should be kept.
 
-The datatype, fieldtype and matchtype have specific values that are allowed.
+The dataType, fieldType and matchType have specific values that are allowed.
```

Comment:
`dataType`, `fieldType` and `matchType` only allow for specific values. A list of these allowed values can be found [here](#allowed-values).